src/utils/tests/json.rs

```diff
@@ -0,0 +1,88 @@
+use crate::utils::json::default_if_null;
+use serde::Deserialize;
+
+
+#[derive(PartialEq, Deserialize, Debug)]
+struct ExpectedStruct {
+    #[serde(deserialize_with = "default_if_null")]
+    field: String,
+    other_field: String,
+    #[serde(deserialize_with = "default_if_null")]
+    with_custom_default: WithCustomDefault,
+}
+
+#[derive(PartialEq, Deserialize, Debug)]
+struct WithCustomDefault(String);
+
+impl Default for WithCustomDefault {
+    fn default() -> Self {
+        WithCustomDefault("custom default".to_string())
+    }
+}
+
+#[test]
+#[should_panic]
+fn deserialize_missing_expected_field() {
+    let test_json = json!({
+        "other_field":"other value"
```

Comment:
Ohhh interesting wasn't aware that it will crash if the key is not present

Reply:
Yeah, I figured I wanted to make sure we are not injecting any stuff :)